src/wp-admin/includes/schema.php

```diff
@@ -424,7 +424,7 @@ function populate_options( array $options = array() ) {
 		'rss_use_excerpt'                 => 0,
 		'mailserver_url'                  => 'mail.example.com',
 		'mailserver_login'                => 'login@example.com',
-		'mailserver_pass'                 => 'password',
+		'mailserver_pass'                 => '',
```

Comment:
Do you think it is worth stripping out each of these and just leaving the port populated.

The mail server and port could probably do with some helper text too:

```diff
diff --git a/src/wp-admin/options-writing.php b/src/wp-admin/options-writing.php
index 21adf262f8..d49f45c341 100644
--- a/src/wp-admin/options-writing.php
+++ b/src/wp-admin/options-writing.php
@@ -161,9 +161,10 @@ if ( apply_filters( 'enable_post_by_email_configuration', true ) ) {
 <table class="form-table" role="presentation">
 <tr>
 <th scope="row"><label for="mailserver_url"><?php _e( 'Mail Server' ); ?></label></th>
-<td><input name="mailserver_url" type="text" id="mailserver_url" value="<?php form_option( 'mailserver_url' ); ?>" class="regular-text code" />
+<td><input name="mailserver_url" type="text" id="mailserver_url" value="<?php form_option( 'mailserver_url' ); ?>" class="regular-text code" aria-describedby="mailserver_url_help" />
 <label for="mailserver_port"><?php _e( 'Port' ); ?></label>
-<input name="mailserver_port" type="text" id="mailserver_port" value="<?php form_option( 'mailserver_port' ); ?>" class="small-text" />
+<input name="mailserver_port" type="text" id="mailserver_port" value="<?php form_option( 'mailserver_port' ); ?>" class="small-text"  aria-describedby="mailserver_port_help" />
+<p><span id="mailserver_url_help"><?php _e( 'Your email provider will provide you with this address, for example <kbd>mail.example.com</kbd>.' ); ?></span> <span id="mailserver_port_help"><?php _e( 'In most cases the port number will be <kbd>110</kbd>.' ); ?></span></p>
 </td>
 </tr>
 <tr>
```

Reply:
Maybe.

[Trac 22942](https://core.trac.wordpress.org/ticket/22942) suggested deprecating Post via email, which makes me think that keeping this simple is likely better. Of course, the deprecation has not had recent progress, and the fields should remain available to anyone who activates the feature afterward.

Having a value of `mail.example.com` would be a good indication that the site does not use it, though checking for either `mail.example.com` **or** an empty `mailserver_url` option is not much more to add in an `upgrade_` function.

If the descriptions are in the same paragraph, the proximity of the `mailserver_url` field and its description is less than ideal with the Port field between them on smaller screens (and/or high zoom). However, it's not very far either.
![both input field descriptions are below the Port field on smaller screens](https://github.com/user-attachments/assets/cb487e96-d383-44de-9c0e-2cd7b54f9ec7)

Emptying the login email address and/or editing its label might also help with [Trac 61702](https://core.trac.wordpress.org/ticket/61702).

Reply:
In wp-mail.php` it's assumed the feature is turned off if the option is empty, I think that's the only place that needs to check.

https://github.com/WordPress/wordpress-develop/blob/8161a8c62a2bfdee02fdbeb4aa7ea773f42e7afa/src/wp-mail.php#L18-L22

That said, yes, let's keep it simple for now.